bfsdk/include/bfsupport.h

```diff
@@ -106,6 +106,22 @@ struct section_info_t {
     uint64_t debug_ranges_size;
 };
 
+/**
+ * @struct efi_data_t
+ *
+ * Binary interface for relaying EFI-related information and desired behavior
+ *
+ * 16 bytes long
+ *
+ */
+#pragma pack(push,8)
+struct efi_data_t {
+    /// Enable EFI exit handlers
```

Comment:
Each field should be documented in the above comment block using the "var" syntax. See other files for examples